.github/workflows/test-workflow.yml

```
@@ -5,33 +5,33 @@ on:
-        description: 'Run Pattern'
+        description: "Run Pattern"
         required: false
         type: string
   workflow_call:
     inputs:
       Timeout:
-        description: 'Timeout for the action in minutes'
+        description: "Timeout for the action in minutes"
         required: true
-        default: '5'
+        default: "5"
```

actionlint (1.7.7)

input "Timeout" of workflow_call event has the default value "5", but it is also required. if an input is marked as required, its default value will never be used

(events)